app/src/main/kotlin/com/wire/android/ui/userprofile/common/UserProfileInfo.kt

```diff
@@ -258,7 +258,7 @@ fun UserProfileInfo(
                         bottom.linkTo(displayName.bottom)
                     }
             ) {
-                if (QRCodeEnabled && isLoading.not()) {
+                if (QRCodeEnabled && showQrCode && isLoading.not()) {
```

Comment:
```suggestion
                if (showQrCode && isLoading.not()) {
```

I think is a good opportunity to remove this FeatureFlag from `FeatureVisibilityFlags`, I remember we were using this while developing but not anymore.